unit_tests/testStrings/testStrings.f90

```diff
@@ -81,6 +83,30 @@ PROGRAM testStrings
   testString2=testString
   ASSERT(CHAR(testString2) == 'testString1','testString2=testString')
 !
+!Test assigning a short integer to string
+  COMPONENT_TEST('Assignment 32-bit Integer to String')
+  testSNK=1_SNK
+  testString=testSNK
+  ASSERT(CHAR(testString) == '1','testString=testSNK : 1')
+  testSNK=-2147483647_SNK
+  testString=testSNK
+  ASSERT(CHAR(testString) == '-2147483647','testString=testSNK : lowest')
+  testSNK=2147483647_SNK
+  testString=testSNK
+  ASSERT(CHAR(testString) == '2147483647','testString=testSNK : highest')
+!
+!Test assigning a long integer to string
+  COMPONENT_TEST('Assignment 64-bit Integer to String')
+  testSLK=1_SLK
+  testString=testSLK
+  ASSERT(CHAR(testString) == '1','testString=testSLK : 1')
+  testSLK=-9223372036854775807_SLK
+  testString=testSLK
+  ASSERT(CHAR(testString) == '-9223372036854775807','testString=testSLK : lowest')
+  testSLK=9223372036854775807_SLK
```

Comment:
Instead of using the largest value use `HUGE(testSLK)` and `-HUGE(testSLK)` on RHS of assignment.